### PR TITLE
[14.0] [IMP] sale_discount_display_amount: add total without discount in report

### PR DESCRIPTION
**Pull request by renda-dev**

### CODE REVIEW

**Review by francesco-ooops — COMMENTED** [review]
Functional ok! 

After enabling the setting in sales, the report will display both discount amount and amount excluding discount.

**Review by francesco-ooops — APPROVED** [review]

**Review by geomer198 — APPROVED** [review]
Code Review ok!

**Comment by francesco-ooops** [comment]
@OCA/crm-sales-marketing-maintainers can we merge?

**Comment by francesco-ooops** [comment]
@pedrobaeza go?

File: sale_discount_display_amount/report/sale_report_template.xml [hunk 1]
```diff
@@ -5,12 +5,26 @@
...
-            </tr>
+            <t groups="product.group_discount_per_so_line" t-if="doc.discount_total">
+                <tr class="border-black">
+                    <td name="td_discount_total_label">
+                        <strong>Discount</strong>
+                    </td>
+                    <td name="td_discount_total" class="text-right">
+                        <span t-field="doc.discount_total" />
+                    </td>
+                    <tr
```
Comment (pedrobaeza) [inline sale_discount_display_amount/report/sale_report_template.xml]: A `<tr>` inside another `<tr>` is incorrect.

**Comment by francesco-ooops** [comment]
@pedrobaeza good now? :)

**Pull request merged**